sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_internal/multi_api.py

```diff
@@ -0,0 +1,50 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from collections import namedtuple
```

Comment:
If you decide to remove the multi api code from the client base, we can get rid of this file too